src/collections/_documentation/development/sdk-dev/unified-api.md

```diff
@@ -54,7 +54,15 @@ meant that certain integrations (such as breadcrumbs) were often not possible.
 
 - **integration**: Code that provides middlewares, bindings or hooks into certain frameworks or environments, along with code that inserts those bindings and activates them. Usage for integrations does not follow a common interface.
 
-- **event processors:** Are callbacks that run for every event. They can either return a "new" event which in most cases means just adding data OR return `null` in case the event will be dropped and not sent.
+- **event processors**: Are callbacks that run for every event. They can either return a "new" event which in most cases means just adding data OR return `null` in case the event will be dropped and not sent.
+
+- **disabled SDK**: Before `init` has been called, or when no client is active,
```

Comment:
`when no client is active,` is ambiguous.

What we have is a "current hub" and each hub has a stack of clients (and scopes).

Here `active` probably refers to `no client on the top of the hub stack`, though maybe we never talk about hub stacks.

Reply:
How would you phrase that ideally? since all the APIs essentially end up at a Hub, which may not have an active Client.

Reply:
```suggestion
- **disabled SDK**: Before `init` has been called or when the current hub has no client
```

?

Perhaps not ideal yet...